data/defaults.yaml

```diff
@@ -337,3 +337,19 @@ prometheus::grok_exporter::package_ensure: 'latest'
 prometheus::grok_exporter::package_name: 'grok_exporter'
 prometheus::grok_exporter::user: 'grok-exporter'
 prometheus::grok_exporter::version: '1.0.0.RC4'
+prometheus::ipsec_exporter::download_extension: 'tar.gz'
```

Comment:
can you please move the data from here to the prometheus::ipsec_exporter class? this allows us to document the default values with puppet-strings. I did not have time yet to cleanup the existing defaults.yaml :(

Reply:
i wonder if we should move the other exporter defaults out of there, i made the same mistake in mine because i saw the examples here...